src/accessibility/color_namer.js

```diff
@@ -0,0 +1,694 @@
+/**
+ * @module Environment
+ * @submodule Environment
+ * @for p5
+ * @requires core
+ */
+
+import p5 from '../core/main';
+import color_conversion from '../color/color_conversion';
+
+let oghsv;
+
+const xcp = [
```

Comment:
thinking of Lauren's comment about the developer docs, it's probably worth a note there (with a short note here) about where these names come from, and whether or not someone can add colors.

Reply:
Just added this to the contributor docs!